dispatchers_test.go

```diff
@@ -23,7 +23,7 @@ func TestDispatchArgDefaultBuiltins(t *testing.T) {
 	localspec := platforms.DefaultSpec()
 	expectedArgs := []string{
 		"BUILDARCH=" + localspec.Architecture,
-		"TARGETPLATFORM=" + localspec.OS + "/" + localspec.Architecture,
+		"TARGETPLATFORM=" + platforms.DefaultString(),
```

Comment:
Should the equivalent in [dispatchers.go](https://github.com/openshift/imagebuilder/blob/8de717452756bdf0348f0393d1205b095d80bdb8/dispatchers.go#L35) be changed too?  IDK for sure, but thought I'd ask.

Reply:
Possibly, but then https://github.com/openshift/imagebuilder/blob/8de717452756bdf0348f0393d1205b095d80bdb8/dispatchers.go#L45-L50 can probably go away as well. Not sure what other consequences this would have on the testsuite and elswhere.